src/pal/inc/mbusafecrt.h

```diff
@@ -32,11 +32,11 @@ typedef int errno_t;
 #define SAFECRT_SUCCESS 0
 
 #ifndef THROW_DECL
-#if defined(_MSC_VER) || defined(__llvm__) || !defined(__cplusplus)
+#if !defined(__cplusplus)
```

Comment:
Why you are changing this for Visual C++ / llmv?

Reply:
I was trying to eliminate compiler differences in the code. Do you have concerns?

Reply:
I was trying to eliminate compiler differences in the code. Do you have concerns?

Reply:
Yes. Now we are adding throw() to functions that were not marked as throw() before for clang / VC++. That may lead to codegen / EH info changes. I would prefer keeping the changes so that they don't introduce differences for clang / VC++.

Reply:
I see, how about renaming the macro to THROW_GCC.

I am worried that someone will use the macro to actually throw an exception and it won't work.

Reply:
This macro doesn't cause any throwing. It just marks a function to indicate that it can throw an exception. The compiler can use it as a hint.

Reply:
Ok, I will leave it alone as long as implication is understood.